.env.example

```
@@ -1,4 +1,6 @@
 SUPABASE_URL=<YOUR_SUPABASE_URL>
 SUPABASE_ANON_KEY=<YOUR_SUPABASE_ANON_KEY>
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)